packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1794,7 +1812,8 @@ class PointerScrollEvent extends PointerSignalEvent with _PointerEventDescriptio
     super.position,
     this.scrollDelta = Offset.zero,
     super.embedderId,
-  });
+    required RespondPointerEventFn respond,
```

Comment:
Should we leave this as optional? May make it easier to remove later on if you're not required to provide a value here e.g. when instantiating one for tests.